extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -335,7 +335,7 @@ export class AmpGeo extends AMP.BaseElement {
             // Only include amp state if user requests it to
             // avoid validator issue with missing amp-bind js
             if (config['AmpBind']) {
-              const geoState = doc.getElementById(GEO_ID);
+              const geoState = ampdoc.getElementById(GEO_ID);
```

Comment:
doc and ampdoc is a bit confusing here. Can we get rid of doc, and use `ampdoc` and `ampdoc.win.document.createElement()` instead? Thank you

Reply:
I appreciate the review. I have completed these changes locally and am running tests on them. When verified working well, I'll add them to this PR.